Umweltzone/src/main/res/values-pt/strings.xml

```diff
@@ -0,0 +1,327 @@
+
+    <!-- AboutActivity / License -->
+    <string name="appinfo_license_title">Licenças</string>
+
+    <string name="appinfo_license_content">
+        O código-fonte do aplicativo Umweltzone é licenciado sob a licença GPLv3.
+        As coordenadas geométricas retiradas do OpenStreetMap são licenciadas sob a
+        Licença Open Database (ODbL) v1.0. Outras coordenadas geométricas são licenciadas e
+        definidas pelas fontes de dados mencionadas. Imagens e textos são licenciados sob
+        a Licença Creative Commons Attribution 3.0 Unported.
+    </string>
+    <string name="appinfo_license_url_title_gpl">Licença GPLv3</string>
+    <string name="appinfo_license_url_title_creative_commons">Licença Creative Commons</string>
+
+    <!-- AboutActivity / Build Info -->
+    <string name="appinfo_build_info_title">Informações da Build</string>
+    <string name="appinfo_build_info_time">Data da Build</string>
+    <string name="appinfo_build_info_version_code">Versão do código</string>
+    <string name="appinfo_build_info_hash">Hash da Build</string>
+
+    <!-- FeedbackActivity / Contact -->
+    <string name="appinfo_contact_title">Contato</string>
+    <string name="appinfo_contact_content">
+        Todos os tipos de feedback, como ideias de melhoria, relatos de erros
+        ou até mesmo solicitações de novas funcionalidades podem ser enviadas para o seguinte endereço:
+    </string>
+
+    <!-- FeedbackActivity / Rating -->
+    <string name="appinfo_rating_title">Avaliação</string>
+    <string name="appinfo_rating_content">
+        Nós ficaríamos muito felizes em receber sua avaliação positiva na Google PlayStore.
+    </string>
+    <string name="appinfo_rating_url_title">Avalie o Umweltzone na Google Play Store</string>
+
+    <!-- DetailsActivity formerly CityInfoFragment -->
+    <string name="city_info_activity_title">Informações da cidade</string>
+    <string name="city_info_section_title_lez">Detalhes sobre as zonas de baixa emissão</string>
+    <string name="city_info_section_title_dpz">Detalhes sobre as zonas de proibição de diesel</string>
+    <string name="city_info_section_title_other">Mais informações</string>
+    <string name="city_info_zone_status">disponível</string>
+
+    <string name="city_info_zone_number_as_of">
+        A partir de <xliff:g example="1 de Out de 2099" id="date">%1$s</xliff:g> apenas veículos com um adesivo
+        <xliff:g example="na cor verde" id="with_colors">%2$s</xliff:g> têm permissão
+        para transitar em zonas de baixa emissão.
+    </string>
+    <string name="city_info_zone_number_since">
+        Desde <xliff:g example="1 de Fev de 2016" id="date">%1$s</xliff:g> apenas
+        veículos com adesivos <xliff:g example="na cor verde" id="with_colors">%2$s</xliff:g> têm permissão
+        para transitar em zonas de baixa emissão.
+    </string>
+
+    <string name="city_info_zone_number_since_date_format">d de MMM de yyyy</string>
```

Comment:
This crashes the app because this must be a formatting string which defines the country-specific date formatting. Legal characters are `d` for day, `M` for month and `Y` for year. See also: [Java 8 documentation](https://docs.oracle.com/javase/8/docs/api/java/text/SimpleDateFormat.html).

Reply:
Ohh Okay. I couldn't test it out due to not being able to completely run the app.
I'm gonna change to d/MM/YYYY, that is another format we use.